helm/hpcc/templates/_helpers.tpl

```diff
@@ -689,13 +689,13 @@ Check that the storage and spill planes for a component exist
 Add command for a component
 */}}
 {{- define "hpcc.componentCommand" -}}
-{{- if .me.valgrind -}}
+ {{- if .me.valgrind -}}
```

Comment:
We don't normally indent the root statements inside a define.  Was this deliberate?

Reply:
you're right we don't (it looks clearer, but..), I'll revert.